src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id?: number;
```

Comment:
Consider making the `id` property non-optional if all todos are expected to have an `id`. This can help prevent runtime errors in components that assume `id` is always present.